### PR TITLE
OSDOCS-8966 updating 4.15 RNs to K8s 1.28 info

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue: [OSDOCS-8966](https://issues.redhat.com/browse/OSDOCS-8966)

Link to docs preview:

QE review:
- [X] QE has approved this change.

Additional information:
Also deleting a section that is no longer needed.

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -680,17 +680,6 @@ In the following tables, features are marked with the following statuses:
 
 {product-title} {product-version} has removed support for the `OPENSHIFT_DEFAULT_REGISTRY` variable. This variable was primarily used to enable backwards compatibility of the internal image registry for earlier setups. The `REGISTRY_OPENSHIFT_SERVER_ADDR` variable can be used in its place.
 
-.APIs removed from Kubernetes 1.27
```
Comment (jeana-redhat) [inline release_notes/ocp-4-15-release-notes.adoc:683]: I know there are none for 1.28, but does it make sense to preserve this table (commented out and empty) for a future release so it's consistent with past versions?

**Review by jeana-redhat — APPROVED** [review]
LGTM, just one philosophical question

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:683]: You are right, that's the better way to go. I'll change it.

Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:696]: @jeana-redhat Changed it to be commented out. Ready for merge.

Comment (jeana-redhat) [reply release_notes/ocp-4-15-release-notes.adoc:696]: will do, whenever the build is done :)

**Pull request merged**